src/web/mjs/connectors/sakuramanga.mjs

```diff
@@ -0,0 +1,82 @@
+import Connector from '../engine/Connector.mjs';
+import Manga from '../engine/Manga.mjs';
+
+export default class SakuraManga extends Connector {
+
+    constructor() {
+        super();
+        super.id = 'sakuramanga';
+        super.label = 'Sakura Manga';
+        this.tags = ['manga', 'japanese'];
+        this.url = 'https://sakuramanga.net';
+
+        this.list = '/japanese-manga-list';
+    }
+
+    async _getMangaFromURI(uri) {
+        try {
+            let request = new Request(uri, this.requestOptions);
+            let data = await this.fetchDOM(request, 'head title');
+            let id = uri.pathname;
+            let title = data[0].text.split(' | ')[0].trim();
+            return new Manga(this, id, title);
+        } catch (error) {
+            return error;
+        }
+    }
+
+    async _getMangaList(callback) {
+        try {
+            let request = new Request(this.url + '/seri-listesi?type=text', this.requestOptions);
```

Comment:
Failed for me locally

<img width="828" alt="manga-list-update" src="https://user-images.githubusercontent.com/27136773/65079839-43fa9080-d9a0-11e9-93df-801d15555d3f.png">

Reply:
facepalm me..... was redoing my original code from what i learned from the tutorial to async, messed up the 
```javascript
let request = new Request(this.url + this.list, this.requestOptions);
```
fix inbound